DogServiceExample/src/test/java/war/DogTest.java

```diff
@@ -1,63 +0,0 @@
-package war;
-
-import java.io.BufferedInputStream;
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.util.Calendar;
-import java.util.Date;
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import junit.framework.Assert;
-import org.junit.Test;
-import app.Dog;
-
-public class DogTest {
-    DateFormat simpleDateFormat = new SimpleDateFormat("dd.MM.yyyy");
-    Calendar c = Calendar.getInstance();
-    String dateToSubmit;
-
-    @Test
-    public void setNameForDog() {
-        Dog dog1 = new Dog();
-        String tempName = dog1.getName();
-        dog1.setName(""); //try to set empty
-        Assert.assertTrue(tempName == dog1.getName());
```

Comment:
Use `assertEquals`.

Also in java `==` is true only if objects are the same instance, may be false even for identical strings.